scripts/eqtl_hail_batch/conditional_analysis.py

```diff
@@ -316,43 +330,37 @@ def spearman_correlation(df):
     # for front-end analysis
     adjusted_spearman_df = t.to_pandas()
     # add celltype id
+    celltype_id = celltype.lower()
+    adjusted_spearman_df['cell_type_id'] = celltype_id
+    # add association ID annotation after adding in alleles, a1, and a2
+    adjusted_spearman_df['association_id'] = adjusted_spearman_df.apply(
+        lambda x: ':'.join(
+            x[['chrom', 'bp', 'a1', 'a2', 'gene_symbol', 'cell_type_id', 'round']]
+        ),
+        axis=1,
+    )
+    adjusted_spearman_df['variant_id'] = adjusted_spearman_df.apply(
+        lambda x: ':'.join(x[['chrom', 'bp', 'a2']]), axis=1
+    )
+    adjusted_spearman_df['snp_id'] = adjusted_spearman_df.apply(
+        lambda x: ':'.join(x[['chrom', 'bp', 'a1', 'a2']]), axis=1
+    )
```

Comment:
Didn't we want to remove those IDs?

Reply:
Sorry yes, I forgot about those. That's removed now